### PR TITLE
Settings improvements

### LINKED ISSUE #10988: Password field in login window get periodically defocused on second attempt.
When I try to log in and put the wrong password first time, second time the password field is periodically defocused, what makes writing password very unpleasant.

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
- Address https://github.com/enso-org/cloud-v2/issues/1457
  - Show x instead of a circle for close tab icon on Windows and Linux
  - Free / Solo no longer shows Organization, Members, User Groups, and Activity Log
  - Invite button should only be there for the admin in Team or above (should already be present)
  - Should list the required symbols in the password change.
  - Changing headshot appeared briefly and then disappeared.
- Fix https://github.com/enso-org/cloud-v2/issues/1455
  - The `Alert` component no longer focuses itself on every keystroke.
- Fix #10988
  - Same issue as above.

Other related changes:
- Fix word wrapping in tooltips when a word is wider than the entire tooltip
- Add word wrapping to "members" table in members tab
- Limit length of "email" column of members table in "user groups" tab
- Hide empty sections
- Add text to empty "user groups" table in "user groups" tab
- ~~Close "set organization name" modal when organization name is set~~
  - Already fixed by another PR

### Important Notes
None

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] ~~Unit tests have been written where possible.~~

### CODE REVIEW

**Comment by somebody1234** [comment]
pinging @MrFlashAccount for CR and @PabloBuchu for QA

**Review by PabloBuchu — APPROVED** [review]

**Comment by PabloBuchu** [comment]
LGTM ✅

**Comment by somebody1234** [comment]
pinging @MrFlashAccount for CR

File: app/ide-desktop/common/src/services/Backend.ts [hunk 1]
```diff
@@ -505,6 +505,21 @@ export interface CreateCustomerPortalSessionResponse {
   readonly url: string | null
 }
 
+/** Whether the user is in an organization. */
+export function isUserInOrganization(user: User) {
```
Comment (MrFlashAccount) [inline app/ide-desktop/common/src/services/Backend.ts]: The name is misleading as it checks if a user has a subscription, but having a subscription is not sufficient to be in an organization, e.g. when the user canceled a subscription

**Review by MrFlashAccount — APPROVED** [review]
LGTM except one comment down below

**Pull request merged**